client/src/components/List.js

```diff
@@ -1,11 +1,8 @@
-import React from "react";
-import PropTypes from "prop-types";
+import styled from "@emotion/styled";
 
-function List({ children }) {
-  return <div>{children}</div>;
-}
+const List = styled.div`
+  width: 90%;
+  overflow: scroll;
+`;
```

Comment:
You're missing a height. If you set a height value the scrollbar will appear ;)

Reply:
This has no effect on my component :/